src/equisolve/utils/convert.py

```diff
@@ -35,17 +36,27 @@ def ase_to_tensormap(
     if not isinstance(frames, list):
         frames = [frames]
 
-    values = [f.info[energy] for f in frames]
+    if energy is not None:
+        values = [f.info[energy] for f in frames]
+    else:
+        energy = "energy"
+        values = [f.get_potential_energy() for f in frames]
 
     if forces is not None:
         positions_gradients = [-f.arrays[forces] for f in frames]
     else:
-        positions_gradients = None
+        try:
+            positions_gradients = [-f.get_forces() for f in frames]
+        except ase.ase.calculators.calculator.PropertyNotImplementedError:
+            positions_gradients = None
```

Comment:
I think you need to check before if there is a calculator otherwise you get  `RuntimeError: Atoms object has no calculator.`

Also if there is a possibility I rather check if something is available in an if statement rather than do a try-except. 

```suggestion
        if (frames[0].get_calculator() is not None) and ("forces" in frames[0].get_calculator().implemented_properties):
            positions_gradients = [-f.get_forces() if (f.get_calculator() is not None) and ("forces" in f.get_calculator().implemented_properties)
                                                   else raise AttributeError(f"First frame has calculator with forces, so assumed all frames have forces but frame {i} does not have forces. Please add or remove forces as property everywhere.")
                                      for i, f in enumerate(frames)]
        else:
               positions_gradients = None
          
```

Its a bit weird logic with the first frame. Thought about something that is a bit more transparent to the user and does not raise an error if the user does not care.
```suggestion
        # consistency check for ase frames 
        has_forces = [(f.get_calculator() is not None) and ("forces" in f.get_calculator().implemented_properties) for f in frames]
        all_same = np.all(has_forces)
        any_true = np.any(has_forces)
        if any_true and all_same:
            positions_gradients = [-f.get_forces() for f in frames]
        elif any_true and not(all_same):
            warnings.UserWarning("The given list of frames contains frames with forces and without forces. We can only create a TensorMap with positions gradients when all frames have set forces. TensorMap will not contain any positions gradients.")
            positions_gradients = None
        else:        
            positions_gradients = None
```

Seems a bit overengineered, but I think it is important to this transparently to the user. In the case where the user gives a string, the user will get an error if not all frames contain the key, that is okay because the user gave the string. In this case, we do stuff even though the user does not give any argument, that is why it needs to be differently handled.